packages/components/color-picker/src/color-picker.vue

```diff
@@ -246,7 +251,7 @@ function handleTrigger() {
 }
 
 function handleConfirm() {
-  color.fromString(customInput.value)
+  debounceSetColorFromString(customInput.value)
```

Comment:
Maybe we just need to judge here, if `customInput.value` has no value, no operation. 🤔
```suggestion
  if (!customInput.value) return
```

Reply:
![image](https://github.com/element-plus/element-plus/assets/17810772/4f7abf81-3ac6-4a2b-b9a1-90d30c5e885a)

实际上用户删除input输入之后，点击可视区域，会有三种触发函数的方式
1.用户点击input内部，不会触发这2个函数
2.用户点击不包含蓝色与红色区域，会触发一个handleConfirm
3.用户点击红色区域，会触发2个
如果按照你说的那种方式的话，用户删除input，然后点击ok按钮不会有任何变化，但是之前的功能是删除input之后，点击ok按钮会变成红色。
ps：区域只是一个粗略的示意图。

Reply:
I think it is not a reasonable phenomenon to turn red, either take an empty string or take the value before it is cleared. What do you think? @ryuhangyeong @chenxch @tolking

Reply:
![image](https://github.com/element-plus/element-plus/assets/17810772/0e10c6c3-a869-41bf-b636-01ef991b9cae)
如果是这样的话，可以把段代码删除了。防抖代码建议还是保留。帮忙看看，幸苦了。

Reply:
Don't worry, maybe we need to discuss with others what is the correct behavior.